Makefile

```diff
@@ -1,4 +1,5 @@
 TEST_OPTIONS=-m unittest discover --start-directory tests --top-level-directory .
+CPU_COUNT=$(shell python -c "from multiprocessing import cpu_count; print(cpu_count())")
```

Comment:
Ahhh, I see what your concern is. :smile: Interestingly, this is a case where Python 2 is forward compatible with Python 3, but not the reverse.